### ISSUE TITLE
Questions about the noise sampling.

### ISSUE THREAD

**Issue by Pandint** [pixeli99/SVD_Xtend#18]
Thank you for sharing! This code helps me a lot.

When using this code to finetune SVD, I have some questions with the noise sample. The noise sampling in this code is as following:
`sigmas = rand_cosine_interpolated(shape=[bsz,], image_d=image_d, noise_d_low=noise_d_low, noise_d_high=noise_d_high,
                                                  sigma_data=sigma_data, min_value=min_value, max_value=max_value).to(latents.device)`
`sigmas = sigmas[:, None, None, None, None]`
`noisy_latents = latents + noise * sigmas`

I want to know if I can replace this simply with a diffusers noise scheduler such as DDPMScheduler?

Hope to get your help!

**Comment by pixeli99** [comment]
In fact, it is feasible, but this will lead to a lot of unnecessary trouble, you need to invest more computing power to do the FT, you can refer to the description in the SVD paper, where they have used the EDM-framework.
![image](https://github.com/pixeli99/SVD_Xtend/assets/46072190/68241e02-2545-451e-89fe-d384234ebbd8)

**Comment by Pandint** [comment]
Thanks a lot!

**Comment by jayhsu0627** [comment]
Hi @pixeli99 , thanks for your great work!

I have a quick question regarding your inference pipeline. I see that your training loop used the modified version of the framework, same as SVD, but why you didn't change the SVD inference pipeline. According to my test, the noise scheduler from your code can generate a offset timestep from [-1.0068533 1.3501322]. 
but the EDM scheduler would create timesteps as:
```
[ 1.6378  1.5755  1.511   1.444   1.3743  1.3018  1.226   1.1469  1.064
  0.9771  0.8855  0.7889  0.6867  0.5781  0.4623  0.3383  0.2048  0.0604
 -0.0971 -0.2702 -0.4622 -0.678  -0.9242 -1.2108 -1.5537]
```
